code/lib/cli/src/repro-templates.ts

```diff
@@ -271,6 +295,7 @@ export const pr: TemplateKey[] = [
   'svelte-vite/default-ts',
   'svelte-kit/skeleton-ts',
   'nextjs/default-ts',
+  'html-vite/default-ts',
```

Comment:
I think we should not add the template to any CI cadences until this PR is merged, we will just test manually, then we can add it after this is merged and the template generation has run.

Reply:
We should probably tweak this file with these details too @yannbf : https://github.com/storybookjs/storybook/blob/next/docs/contribute/code.md#developing-a-template